libr/debug/p/debug_native.c

```diff
@@ -218,7 +218,12 @@ static int r_debug_native_stop(RDebug *dbg) {
 }
 
 /* TODO: specify thread? */
-/* TODO: must return true/false */
+
```

Comment:
Remove the comment, its unnecessary to document internal functions and i think the current signature is clear enough

Reply:
Yes indeed if this same function is used for each module, I should not document each.